index.html

```diff
@@ -0,0 +1,247 @@
+                        </div>
+                    </div>
+                </div>
+
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project4.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Multi-Post Stories</h1>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill set1">html</li>
+                            <li class="skill set2">css</li>
+                            <li class="skill set3">javascript</li>
+                        </ul>
+                        <div class="button-click">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section id="about" class="about">
+            <div class="left">
+                <h2 class="about-me"> About Myself</h2>
+                <p class="intro1">
+                    Hello I’m a software developer! I can help you build a
+                    product , feature or website Look through some of my
+                    work and experience! If you like what you see and have
+                    a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h3 class="intro2">LET’S CONNECT</h3>
+                <ul class="social">
+                    <li class="social-icon">
+                        <img src="images/github.png" alt="github">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/Linkedin.png" alt="Linkedln">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/fingers.png" alt="two fingers">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/twitter.png" alt="twitter">
+                    </li>
+
+                    <li class="social-icon">
+                        <img src="images/m.png" alt="m">
+                    </li>
+                </ul>
```

Comment:
- [ ] The about section in the Figma design has a round corner at the top right side as marked in the image below. Please add this to your project. There is more padding between the first text inside the about section. Ensure that you use the exact margin, padding and spaces as seen on figma design.
- [ ] This about section is not well aligned in the center as seen in the image below. 

Figma design
![image](https://user-images.githubusercontent.com/84156714/228537150-fee121ac-cbc6-49ca-83cf-7930441843d3.png)

Your work
![image](https://user-images.githubusercontent.com/84156714/228537306-d0585e76-0be9-42aa-be98-9c7b42901a0d.png)